### PR TITLE
KEYCLOAK-6818 Extra AUTH_SESSION_ID cookie with a path of /auth

### PR DESCRIPTION
**Pull request by martin-kanis**

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
LGTM

Adding also @hmlnarik and @matthewhelmke for review.

File: server_installation/topics/clustering/sticky-sessions.adoc [hunk 1]
```diff
@@ -144,6 +144,8 @@ In the undertow subsystem, add the `session-config` under the `servlet-container
 </servlet-container>
 ----
 
+WARNING: Use the `session-config` only with mod_cluster load balancer. In other cases configure sticky session over the `AUTH_SESSION_ID` cookie directly in a load balancer.   
+
```
Comment (matthewhelmke) [inline server_installation/topics/clustering/sticky-sessions.adoc]: A couple small edits:

WARNING: Use the `session-config` only with the mod_cluster load balancer. In other cases, configure sticky session over the `AUTH_SESSION_ID` cookie directly in a load balancer.

File: server_installation/topics/clustering/sticky-sessions.adoc [hunk 2]
```diff
@@ -169,3 +171,5 @@ cd $RHSSO_NODE2
 
 Access the server on `http://localhost:8080/auth` . Creation of admin user is possible just from local address and without load balancer (proxy) access,
 so you first need to access backend node directly on `http://localhost:8180/auth` to create admin user.
+
+WARNING: You should always access the cluster through load balancer and not through backend node directly, when using mod_cluster.
```
Comment (matthewhelmke) [inline server_installation/topics/clustering/sticky-sessions.adoc]: Minor edits:
WARNING: When using mod_cluster you should always access the cluster through the load balancer and not directly through the backend node.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
I have suggested some language-related edits.

**Comment by martin-kanis** [comment]
@matthewhelmke fixed, thank you

**Review by matthewhelmke — APPROVED** [review]

**Review by hmlnarik — APPROVED** [review]
LGTM, thanks!

**Pull request merged**